<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogSearchAccount</class>
 <widget class="QDialog" name="DialogSearchAccount">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>517</width>
    <height>370</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ricerca conti</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="lblInfo1">
     <property name="text">
      <string>Attraverso i campi di ricerca sottostanti è possibile filtrare i conti visualizzati.</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="grpBoxDataSearch">
     <property name="title">
      <string>Campi di ricerca</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="lblName">
        <property name="text">
         <string>Nome:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLineEdit" name="txtName">
        <property name="text">
         <string>*</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="lblSurname">
        <property name="text">
         <string>Cognome:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLineEdit" name="txtSurname">
        <property name="text">
         <string>*</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="lblBalance">
        <property name="text">
         <string>Saldo:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="cmbBalancePattern">
        <item>
         <property name="text">
          <string>*</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>=</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>&gt;</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>&gt;=</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>&lt;</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>&lt;=</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QLineEdit" name="txtBalance">
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="btnSearch">
        <property name="text">
         <string>Cerca</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnResetFields">
        <property name="text">
         <string>Resetta campi</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="lblInfo2">
     <property name="text">
      <string>Facendo doppio click su un conto sarà possibile modificarlo, cancellarlo, prelevare o depositare.</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTableWidget" name="tblSearchResult">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>125</number>
     </attribute>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <column>
      <property name="text">
       <string>Tipo</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Nome</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Cognome</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Saldo</string>
      </property>
     </column>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>txtName</tabstop>
  <tabstop>txtSurname</tabstop>
  <tabstop>cmbBalancePattern</tabstop>
  <tabstop>txtBalance</tabstop>
  <tabstop>tblSearchResult</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
